### PR TITLE
SAGA Audio support

### PR DESCRIPTION
**Pull request by MBeijer**

### CODE REVIEW

File: src/tracker/SectionSettings.cpp [hunk 1]
```diff
@@ -744,40 +744,149 @@ class TabPageIO_3 : public TabPage
...
+			screen->paintControl(container->getOwnerControl());
+	}
+
     virtual void init(PPScreen* screen)
     {
         pp_int32 x = 0;
         pp_int32 y = 0;
-        
+
+		this->screen = screen;
```
Comment (MBeijer) [inline src/tracker/SectionSettings.cpp:852]:
```suggestion
        this->screen = screen;
```

File: src/milkyplay/CMakeLists.txt [hunk 2]
```diff
@@ -139,20 +139,31 @@ if(APPLE)
     )
     message(STATUS "Enabled Core Audio support")
 elseif(AROS OR AMIGA)
+    enable_language(ASM_VASM)
```
Comment (MBeijer) [inline src/milkyplay/CMakeLists.txt:142]:
```suggestion
    enable_language(ASM-VASM)
```

File: include/amiga/clib/cybergraphics_protos.h [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/*
```
Comment (MBeijer) [inline include/amiga/clib/cybergraphics_protos.h:1]: system specific headers has no place in the project source tree. Vampire Lib should be included with the gcc6 toolchain, not in all the projects that uses it.

**Comment by MBeijer** [comment]
@andrewoesten have you read my comments?

**Comment by titanlab** [comment]
Created PR #43 in favor of this one.

**Pull request closed without merging**